apps/omg_watcher/lib/omg_watcher/api/status.ex

```diff
@@ -75,7 +77,7 @@ defmodule OMG.Watcher.API.Status do
       last_seen_eth_block_number: eth_block_number,
       last_seen_eth_block_timestamp: eth_block_timestamp,
       eth_syncing: eth_syncing,
-      byzantine_events: events_processor ++ events_block_getter,
+      byzantine_events: events_processor ++ events_block_getter ++ ethereum_height_events,
```

Comment:
I'm guessing this will include both `error` and `stalled` events. Hm, I think ideally, this (`status.get`'s byzantine events list) should only display the most "top-level" event available. So currently `ethereum_sync_stalled` and later on, the generic `stalled_syncing` from #814. The details should then be displayed via `alarm.get`.

But not 100% sure about this approach and this is also pending the UX of byzantine events APIs topic.

Reply:
Ah makes sense. I forgot we have `/alarm.get`.